### DISCUSSION TITLE
Scavenger Hunt Mission: Show us your "view" 🖼

### DISCUSSION THREAD

**Discussion by elisemoe** [education/hackconviii#5] [General]
Share a picture of your “view” at the conference. Laptop on the couch? Maybe the weather's nice and you're hanging out outside? Let's see what HackCon looks like in your slice on the universe.

**Comment by sampoder** [comment]
my messy desk rn!

![20200822_200109](https://user-images.githubusercontent.com/39828164/90955758-b0381b00-e4b2-11ea-9bba-3aab2e59185d.jpg)

**Reply by csleong98** [reply]
those sticker packs though

**Comment by radhikasethi2011** [comment]
My Desk for the evening!
![IMG_20200822_184539](https://user-images.githubusercontent.com/44310177/90956947-8c6fd780-e4a8-11ea-9e61-e446f6a9fec7.jpg)

**Comment by KhanhDoHoang** [comment]
![117972446_3301198583438681_8784149144277329679_n](https://user-images.githubusercontent.com/55791090/90957119-dd0c1400-e459-11ea-8402-bf81ae8b828f.jpg)

**Reply by charliebaer** [reply]
is that a wooden speaker..?

**Reply by KhanhDoHoang** [reply]
It's a phone bender, speaker is just a drawing ^^

**Comment by usamaeltmsah** [comment]
It's afternoon :sun_with_face: 
![image](https://user-images.githubusercontent.com/23380057/90957189-ac919d80-e48c-11ea-9afb-bfa12fbb9a1b.png)

**Comment by charliebaer** [comment]
![Deskmlh](https://user-images.githubusercontent.com/50674268/90957494-83343a00-e4ab-11ea-9917-f55d864a826f.jpg)
I know it's messy..don't cringe

**Comment by csleong98** [comment]
![photo_2020-08-22_22-30-22](https://user-images.githubusercontent.com/6150058/90958491-6e659f80-e4c7-11ea-9c62-fd37bf02445f.jpg)
Hackcon, Tea, Mario and Me

**Comment by jaypatel15406** [comment]
Hot Coffee, GitHub :octocat: & MLH HackCon 👍  
![IMG_20200822_191549](https://user-images.githubusercontent.com/50138179/90959857-250f5300-e4bb-11ea-92cf-70d24541add4.jpg)

**Comment by BazilNizam** [comment]
![photo_2020-08-22_21-01-22](https://user-images.githubusercontent.com/22983667/90959872-407a5e00-e4bb-11ea-9555-735901f896f1.jpg)

**Comment by BazilNizam** [comment]
enjoying hackon

**Comment by mohdsaabith** [comment]
ENJOYING THE MOMENTS !!!
![WhatsApp Image 2020-08-22 at 21 10 23](https://user-images.githubusercontent.com/44156514/90960001-2db45900-e4bc-11ea-9e1e-b4717c080ccb.jpeg)

**Comment by HrithikSahu** [comment]
My First Hackon and i'm Loving it. I have Learned so much in Just one day than I have Learned in my Last Month of Online College Lectures.

![WhatsApp Image 2020-08-22 at 11 45 13 PM](https://user-images.githubusercontent.com/61985253/90963028-32840780-e4d2-11ea-8078-33feaf268496.jpeg)

**Comment by annaarroyo** [comment]
![Image from iOS (5)](https://user-images.githubusercontent.com/43049986/90963902-78fd4600-e481-11ea-95f1-b47b51b268af.jpg)
 
eating my breakfast! i love hackcon so far, even from my dining table :)

**Comment by LilyPerr** [comment]
![IMG_7298](https://user-images.githubusercontent.com/42851016/90964100-8cadaa00-e48b-11ea-8904-293ae4b148b7.JPG)
I always keep my hydro flask near my desk!

**Comment by BStarcheus** [comment]
![IMG_6764](https://user-images.githubusercontent.com/41553156/90964682-d51b9680-e490-11ea-8bbb-7dc280d5280e.JPG)
At least I'm getting a little view of the outside world :)

**Comment by carmenx** [comment]
This is my setup! I have multiple tabs open but at least I have a cute dog tote bag to cheer me on with this scavenger hunt! 
![image](https://user-images.githubusercontent.com/61920742/90965404-a09ddf80-e47c-11ea-8a73-5be3806fa7f7.png)

**Comment by dipakparmar** [comment]
![20200822_202328613_iOS](https://user-images.githubusercontent.com/24366206/90965512-9b8d6000-e47d-11ea-9e6d-83f8bb89ee11.jpg)

**Comment by mike-taylor99** [comment]
![IMG_3129](https://user-images.githubusercontent.com/47544533/90966103-8673f980-e49c-11ea-98f5-c73e698bd045.jpg)

**Comment by Raymo111** [comment]
Here's my desk rn!
![IMG_20200822_182231](https://user-images.githubusercontent.com/34818275/90966771-a0193f00-e4a4-11ea-977c-1b5b4c6cc493.jpg)

**Comment by PaHorton** [comment]
My desk is rather cluttered but I still love my setup!

![image](https://user-images.githubusercontent.com/13230281/90970507-f09b9700-e4ba-11ea-849d-49dc4b8142ee.png)

**Reply by Raymo111** [reply]
love the brick!!

**Comment by Garoto0518** [comment]
![WhatsApp Image 2020-08-23 at 1 29 03 PM](https://user-images.githubusercontent.com/22475754/90984707-b37be700-e544-11ea-8d35-0cf341b4eeec.jpeg)

Ignore the mess; but, We love multi-tasking <3

**Comment by 0xjba** [comment]
![photo6204168942012967633](https://user-images.githubusercontent.com/38902140/90984758-b7375a00-e594-11ea-957e-e64bfe67fe2e.jpg)
My First Hackcon! Oh Yeah <3 it.
`My Borrowed Desktop Setup for Hackcon.

**Comment by surajsahani** [comment]
![IMG_20200824_141549](https://user-images.githubusercontent.com/22853459/91023919-c9f17380-e614-11ea-8412-04ffde8bfa7b.jpg)
Before Hackcon!
![WhatsApp Image 2020-08-22 at 11 24 38 PM](https://user-images.githubusercontent.com/22853459/91023979-e1306100-e614-11ea-9498-45cfc88ba83c.jpeg)
During Hackcon (Avengers Assemble :D)
